pyeudiw/tests/storage/test_db_engine.py

```diff
@@ -131,6 +131,25 @@ def test_update_unexistent_trusted_attestation(self):
         except StorageWriteError as e:
             return
 
+    def test_update_trusted_attestation_metadata(self):
+        replica_count = self.engine.add_trust_attestation_metadata(
+            self.federation_entity_id, {"metadata": "test"})
+        
+        assert replica_count > 0
+        
+        ta = self.engine.get_trust_attestation(self.federation_entity_id)
+
+        assert ta.get("metadata", None) != None
+        assert ta["metadata"] == {"metadata": "test"}
+
+    def test_update_unexistent_trusted_attestation_metadata(self):
+        try:
+            self.engine.add_trust_attestation_metadata(
+                "test", {"metadata": "test"})
```

Comment:
see my prev comment

Reply:
Resolved in [d90974c](https://github.com/italia/eudi-wallet-it-python/pull/188/commits/d90974ce11036794799a476e4307cfd2796149e8)